sgtm/apollo/gtm/variable_template.tpl

```diff
@@ -0,0 +1,257 @@
+﻿___INFO___
+
+{
+  "type": "MACRO",
+  "id": "cvt_temp_public_id",
+  "version": 1,
+  "securityGroups": [],
+  "displayName": "gps-apollo",
+  "description": "Get data from a Google Sheet in realtime. A quick \u0026 lightweight route to get data without needing more complex cloud setups.",
+  "containerContexts": [
+    "SERVER"
+  ]
+}
+
+
+___TEMPLATE_PARAMETERS___
+
+[
+  {
+    "type": "TEXT",
+    "name": "spreadsheet_id",
+    "displayName": "The spreadsheet id that we get the data from.",
+    "simpleValueType": true
+  },
+  {
+    "type": "TEXT",
+    "name": "age",
+    "displayName": "The value of age group that we input in the form.",
+    "simpleValueType": true
+  },
+  {
+    "type": "TEXT",
+    "name": "address",
+    "displayName": "The address that we input in the form.",
+    "simpleValueType": true
+  },
+  {
+    "type": "TEXT",
+    "name": "job",
+    "displayName": "The related job that we input in the form.",
+    "simpleValueType": true
+  }
+]
+
+
+___SANDBOXED_JS_FOR_SERVER___
+
```

Comment:
Please add the license header:

```
// Copyright 2024 Google LLC
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     https://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
//
```

Reply:
Thank you for reviewing!!
I added the license header.

Thanks.